File: doc/how-tos/rotate-key/java/step3.java
		System.out.println("\n9. Generating new Verkey of Trust Anchor in the wallet\n");
		String newTrustAnchorVerkey = Did.replaceKeysStart(walletHandle, trustAnchorDID, "{}").get();
		System.out.println("New Trust Anchor's Verkey: " + newTrustAnchorVerkey);

		System.out.println("\n10. Building NYM request to update new verkey to ledger\n");
		String nymUpdateRequest = buildNymRequest(trustAnchorDID, trustAnchorDID, newTrustAnchorVerkey, null, "TRUST_ANCHOR").get();
		System.out.println("NYM request:\n" + nymUpdateRequest);

		System.out.println("\n11. Sending NYM request to the ledger\n");
		String nymUpdateResponse = signAndSubmitRequest(pool, walletHandle, trustAnchorDID, nymUpdateRequest).get();
		System.out.println("NYM response:\n" + nymUpdateRequest);

		System.out.println("\n12. Applying new Trust Anchor's Verkey in wallet\n");
		Did.replaceKeysApply(walletHandle, trustAnchorDID);

File: doc/how-tos/rotate-key/python/step3.py
		print_log('\n9. Generating new verkey of trust anchor in wallet\n')
		new_verkey = await did.replace_keys_start(wallet_handle, trust_anchor_did, "{}")
		print_log('New Trust Anchor Verkey: ', new_verkey)
        
		print_log('\n10. Building NYM request to update new verkey to ledger\n')
		nym_request = await ledger.build_nym_request(trust_anchor_did, trust_anchor_did, new_verkey, None, 'TRUST_ANCHOR')
		print_log('NYM request:')
		pprint.pprint(json.loads(nym_request))
        
		print_log('\n11. Sending NYM request to the ledger\n')
		nym_response = await ledger.sign_and_submit_request(pool_handle, wallet_handle, trust_anchor_did, nym_request)
		print_log('NYM response:')
		pprint.pprint(json.loads(nym_response))
        
		print_log('\n12. Apply new verkey in wallet\n')
		await did.replace_keys_apply(wallet_handle, trust_anchor_did)
